aict-MERN/backend-of-aict-site-w-react/app.js

```diff
@@ -0,0 +1,24 @@
+import express from 'express'
+const app = express()
+import router from './routes/posts.js'
+import cors from 'cors';
+
+const corsOptions = {
+  origin: (origin, callback) => {
+    callback(null, true);
+  },
+  methods: ["GET", "POST", "PUT", "PATCH", "DELETE"],
+  allowedHeaders: ["Access-Control-Allow-Origin", "Origin", "X-Requested-With", "Content-Type", "Accept", "Authorization"],
+  credentials: true
+};
+
+app.options('*', cors(corsOptions));
+app.use(cors(corsOptions));
+
+app.use(express.json())
+app.use(express.urlencoded({ extended: true }))
+
+app.use('/', router)
+
+const port = process.env.PORT || 3001
```

Comment:
veeery nice to see. the only thing you could add to this is the [dotenv library](https://www.npmjs.com/package/dotenv)

Reply:
Lol that would help. I think its just become second nature for that to be the port variable. Then I forget to import the module